src/libs/kata-types/src/config/hypervisor/mod.rs

```diff
@@ -237,6 +239,18 @@ impl BootInfo {
         Ok(())
     }
 
+    /// Add kernel parameters to bootinfo. It is always added before the original
+    /// to let the original one takes priority
+    pub fn add_kparams(&mut self, params: Vec<String>) {
```

Comment:
Suggestion changing `add_kparams` to `add_kernel_params`, `k` is too short as an abbreviation.

Reply:
Fixed